doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -311,6 +311,32 @@ The following software versions are considered to benchmark Kubernetes v1.21
 | Functest                | v1.21       |
 | xrally-kubernetes       | 1.1.1.dev12 |
 
+### Dataplane benchmarking
+
+[Kubernetes perf-tests](https://github.com/kubernetes/perf-tests) stores
+various Kubernetes-related performance test related tools especially
+[netperf](https://github.com/kubernetes/perf-tests/tree/master/network/benchmarks/netperf)
+which benchmarks Kubernetes networking performance.
+
+As listed in [README](https://github.com/kubernetes/perf-tests/tree/master/network/benchmarks/netperf#readme),
```

Comment:
@collivier should we be clear that this is netperf README?

Reply:
It's no longer asis to take Georg's comments into account. Yes the first patchset was a copy/paste on purpose.